acl_loader/main.py

```diff
@@ -150,7 +187,16 @@ def read_policers_info(self):
         Read POLICER table from configuration database
         :return:
         """
-        self.policers_db_info = self.configdb.get_table(self.POLICER)
+        
+        """
+        For multi-npu platforms we will read from any one of front asic namespace 
+        config db as the information should be same across all config db
+        """
+        if self.per_npu_configdb:
+            namespace_configdb = (self.per_npu_configdb.values())[0]
```

Comment:
similar comment here .. Will the table POLICER have different keys between the TABLE in global and namespace DB ?

Reply:
@judyjoseph global db will not have key on muti-npu platforms and it will same for all front asic namespace.